bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx78-b-de.html

```diff
@@ -0,0 +1,69 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import send_to_device with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
```

Comment:
There's quite a bit of code duplication in the templates here for each variation. As far as I can tell, only the title is changing - so would it be less code to write / maintain if we extended the control template and then update the title using a block?

```jinja
{# This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}

{% from "macros.html" import send_to_device with context %}

{% extends "firefox/whatsnew/whatsnew-fx78-a-de.html" %}

{% block page_heading %}Privat unterwegs? Hol dir Firefox für Mobile.{% endblock %}
```